sklearn/svm/tests/test_svm.py

```diff
@@ -275,9 +275,9 @@ def test_tweak_params():
     """
     clf = svm.SVC(kernel='linear', C=1.0)
     clf.fit(X, Y)
-    assert_array_equal(clf.dual_coef_, [[.25, -.25]])
+    assert_array_equal(clf.dual_coef_, [[-.25, .25]])
     assert_array_equal(clf.predict([[-.1, -.1]]), [1])
-    clf.dual_coef_ = np.array([[.0, 1.]])
+    clf._dual_coef_ = np.array([[.0, 1.]])
```

Comment:
We could forbid overwriting ``dual_coef_`` as we did with ``coef_``. I'm not sure that is necessary, though.